src/main/java/appeng/bootstrap/FeatureFactory.java

```diff
@@ -123,39 +127,32 @@ public FeatureFactory features( AEFeature... features )
 		return new FeatureFactory( this, features );
 	}
 
-	void addBootstrapComponent( IBootstrapComponent component )
+	public void addBootstrapComponent( IBootstrapComponent component )
 	{
-		this.bootstrapComponents.add( component );
+		Arrays.stream( component.getClass().getInterfaces() )
+				.filter( i -> IBootstrapComponent.class.isAssignableFrom( i ) )
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this lambda with a method reference. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS1612)